src/actions/profile-view.js

```diff
@@ -41,6 +41,14 @@ export function changeSelectedThread(selectedThread: ThreadIndex): Action {
 }
 
 export function changeThreadOrder(threadOrder: ThreadIndex[]): Action {
+  const { ga } = window;
+  if (ga) {
+    ga('send', {
+      hitType: 'event',
+      eventCategory: 'profile',
+      eventAction: 'change thread order',
+    });
+  }
```

Comment:
I'm a little against a utility function here. To me the interface feels simple enough to repeat as we need, especially since we have type coverage. If you really feel that it's warranted, I'd prefer to wrap all the `ga` uses up in to a `src/utils/analytics.js` file.